packages/grafana-ui/src/components/DataSourceSettings/types.ts

```diff
@@ -1,5 +1,11 @@
+import React from 'react';
 import { DataSourceSettings } from '@grafana/data';
 
+export interface AzureSettings {
```

Comment:
It feels like this probably should be placed together with the datasource. If we do the refactoring I mentioned above it should be easier to move this.

Do we have any need of sharing this between multiple datasource?

Reply:
I just had a sync with @joshhunt and since we are going to have the backend code as a part of the core I think this is okay to keep in the grafana-ui. So we can ignore this comment.

Reply:
The only thing I would like us to do is to change the name from `AzureSettings` to `AzureAuthSettings`

Reply:
Renamed